source/concepts/faq.md

```diff
@@ -66,12 +66,13 @@ Nixpkgs and NixOS have both stable and rolling releases.
 
 ### Stable
 
-- On Linux (including NixOS and WSL), use [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).
+- On macOS/Darwin, use [nixpkgs-*-darwin](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-)
 
-  These branches only contain commits that passed the NixOS test suite.
-- On any other platform, use [`nixpkgs-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-).
+  These branches point to commits where most Darwin packages can be substituted and don't have to be built.
+- On any other platform, use [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).
```

Comment:
Not sure what is the value of using the NixOS channel by default, most people using Nixpkgs without NixOS won't care about the NixOS test suite.

How do we explain the rationale to someone else?

Reply:
We don't really have a choice right now. There's no stable Hydra channel that contains Linux builds but doesn't run the NixOS tests.

This is pretty much unchanged with this PR though, already before this section recommended the `nixos-*` channels to all Linux users, even if they're not on NixOS.

Reply:
Why do you think https://hydra.nixos.org/job/nixpkgs/trunk/unstable#tabs-constituents does not contain Linux builds?

Reply:
Ah a stable channel.